src/main/kotlin/org/rust/ide/annotator/fixes/ConvertToTyWithDerefsRefsFix.kt

```diff
@@ -25,14 +26,33 @@ data class DerefRefPath(val derefs: Int, val refs: List<Mutability>)
  * The fix applies `path.derefs` dereferences to the expression and then references of the mutability given by
  * `path.refs`.  Note that correctness of the generated code is not verified.
  */
-class ConvertToTyWithDerefsRefsFix(expr: PsiElement, val ty: Ty, val path: DerefRefPath) : LocalQuickFixAndIntentionActionOnPsiElement(expr) {
+class ConvertToTyWithDerefsRefsFix(
+    expr: PsiElement,
+    val ty: Ty,
+    val path: DerefRefPath
+) : LocalQuickFixAndIntentionActionOnPsiElement(expr) {
     override fun getFamilyName(): String = "Convert to type"
 
-    override fun getText(): String = "Convert to $ty using dereferences and/or references"
+    override fun getText(): String = "Convert to $ty by applying ${formatRefs(path)}"
```

Comment:
Let's better use `using` instead of `by applying` because:
* it's shorter
* it's consistent with other fixes